docs/intro.md

```diff
@@ -38,9 +38,9 @@ The JS is also available via npm and can be integrated into your regular build p
 
 ### Editorial Workflow
 
-Netlify CMS has an optional [editorial workflow](https://github.com/netlify/netlify-cms/blob/master/docs/editorial-workflow.md) that translates common Git commands into familiar language in a simple UI:
+Netlify CMS has an optional [editorial workflow](/docs/editorial-workflow.md) that translates common Git commands into familiar language in a simple UI:
```

Comment:
We actually need to remove the `.md` on this as well. It's true that it breaks the link in the GitHub view, but until we have a solution in place to accommodate both, we'll have to put priority on the published site, and it relies on the "pretty" path.